--- speak.pl	2013-10-02 11:26:07.000000000 +0200
+++ speak.pl	2014-08-07 11:05:27.000000000 +0200
@@ -137,13 +137,14 @@
 
 use strict;
 
+use File::Path qw( rmtree );
 use HTTP::Cookies;
 use WWW::Mechanize;
 use LWP;
 use HTML::TreeBuilder;
 use Data::Dumper;
 $Data::Dumper::Maxdepth = 2;
- 
+
 if (scalar(@ARGV) != 3) {
 	print STDERR "Usage: $0 LANGUAGE IN.txt OUT.mp3\n";
 	print STDERR "\n";
@@ -277,6 +278,7 @@
 print "Writing output to $all_mp3_out...";
 JoinMp3s(\@all_mp3s, $all_mp3_out);
 print "done\n";
+rmtree( $TMP_DIR );
 
 sub JoinMp3s() {
 	my $mp3s_ref = shift;
